src/types.ts

```diff
@@ -34,3 +34,8 @@ export type TransitionComponent = FunctionComponent<{
   direction?: 'in' | 'out';
   onTransitionEnd?: (direction: 'in' | 'out') => void;
 }>;
+
+export type Order<Field> = {
+  field: Field;
```

Comment:
From the implementation I see that `Field` is expected to be a string which can index a row object. In that case `Order<FieldName extends string>` would make that clearer.